migrations/README

```diff
@@ -0,0 +1,5 @@
+<<<<<<< HEAD
```

Comment:
It looks like this file (which Alembic should have generated itself) somehow ended up with a merge conflict in it? Not sure what happened here.